src/main/java/appeng/util/Platform.java

```diff
@@ -1431,17 +1431,17 @@ public static void configurePlayer( final EntityPlayer player, final AEPartLocat
 		player.rotationYaw = player.prevCameraYaw = player.cameraYaw = yaw;
 	}
 
-	public static boolean canAccess( final AENetworkProxy gridProxy, final BaseActionSource src )
+	public static boolean canAccess( final AENetworkProxy gridProxy, final IActionSource src )
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Refactor this method to not always return the same value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS3516)